src/views/Swap/Swap.vue

```diff
@@ -725,12 +725,11 @@ export default {
 
       const account = isERC20(this.asset) ? this.account : this.toAccount
       const balance = account?.balances[this.selectedQuote.bridgeAsset]
-      const toSwapFeeInUnits = currencyToUnit(
+      const SwapFeeInUnits = currencyToUnit(
         cryptoassets[this.selectedQuote.bridgeAsset],
-        this.receiveFee
+        isERC20(this.asset) ? this.fromSwapFee : this.receiveFee
       )
-
-      return BN(balance).gt(toSwapFeeInUnits)
+      return BN(balance).gt(SwapFeeInUnits)
```

Comment:
Why is this not camel case anymore?

Reply:
should be with camelCase - correct 
I will fix it in another PR as this was merged already...